### PR TITLE
Add onScrollEndDrag & onMomentumScrollEnd props

### PR DESCRIPTION
**Pull request by SergeiGrigorev**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -46,3 +46,5 @@ dist/
 node_modules/
 yarn.lock
 package-lock.json
+
+.history/
```
Comment (haikov) [inline .gitignore]: Let's not add such things to our forks. Fine to have it in project repos, but if we'd want to sync this with upstream at some point, extensions might not be accepted.

**Review by haikov — APPROVED** [review]
Looking good to me 👍

File: .gitignore (hunk 1 above)
Comment (haikov) [reply .gitignore]: @SergeiGrigorev remove this please before merging.

Comment (SergeiGrigorev) [reply .gitignore]: Done

**Pull request merged**